### ISSUE TITLE
Got error:  Class private methods are not enabled. Please add `@babel/plugin-transform-private-methods` to your configuration.

### ISSUE THREAD

**Issue by a-bochilo** [astoiccoder/gatsby-plugin-google-gtag-cookieconsent#2]
Got this error right after plugin was added. I removed all private # symbols from ast.js file and it started to work in dev. But obviously it won't work in prod. Solution, provided in error descritpion, didn't help. 
"gatsby": "^5.13.6"

![image](https://github.com/astoiccoder/gatsby-plugin-google-gtag-cookieconsent/assets/103776395/a1305dff-c648-4723-8c37-a7be830bdd40)

**Comment by eudamniac** [comment]
Thanks for raising this issue @a-bochilo!

It's hard to know why you are getting this error without knowing more about your project setup. In case it would be possible to have a minimal example to reproduce it, I would be happy to have a look into it. Did you have any babel setup in your main project?

You could also try if the original plugin works for you: https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-google-gtag - it has the same dependency on `minimatch`.

I assume you tried adding this plugin as well https://babeljs.io/docs/babel-plugin-transform-private-methods#usage. For what it's worth, I'm just curious if you also added the option `loose: true`.

Furthermore I updated the dependencies on this plugin, but I wouldn't expect this to have any implication on the error you are experiencing.

**Comment by lymperis-e** [comment]
@a-bochilo I had the same issue. I think it might have to do with CommonJS syntax. I used default exports (the readme suggests named exports) like so:

```javascript
// cookie-consent-config.js
module.exports = {
  categories: {
    necessary: {
     ...
...
```

instead of 
```javascript
// cookie-consent-config.js
exports.cookieConsentConfig = {
  categories: {
    necessary: {
     ...
...
```

Of course I then modified my imports in `gatsby-config.js` to
```javascript
const cookieConsentConfig = require("./cookie-consent-config")
const googleGtagPluginConfig = require("./gtag-config")
```

**Comment by lymperis-e** [comment]
Actually, that is not at all the problem. It seems that #1 is not resolved

**Comment by eudamniac** [comment]
Fixed in version 0.0.6
